app/models/course_data/articles_courses.rb

```diff
@@ -49,7 +49,7 @@ def new_article
   end
 
   def live_manual_revisions
-    course.revisions.live.where(article_id: article_id)
+    course.tracked_revisions.live.where(article_id: article_id)
```

Comment:
This one should still use `revisions` instead of `tracked_revisions`, I think. We mark an article as untracked on teh ArticlesCourses record itself, but there's no reason not to update the data on the AC record, even if it's being excluded from other stats.